app/views/layouts/welcome.html.erb

```diff
@@ -5,6 +5,9 @@
 
     <%= render HeaderComponent.new %>
 
+    <%# FIXME: remove this to enable the page in prod %>
```

Comment:
@ethax-ross can you confirm this is all good form a dev perspective to stop it rendering in prod? I'll review the rest of the copy and design in the wider PR.

Reply:
I would have stuck in a route to catch it but that'll get the job done! 🙈

Reply:
Yeah, route would be much cleaner but went for the keeping it all in one place approach 😅